shotover/src/transforms/kafka/sink_cluster/mod.rs

```diff
@@ -1548,6 +1698,7 @@ impl KafkaSinkCluster {
                 ))
             }
         }
+        base.invalidate_cache();
```

Comment:
why do we need 2 `base.invalidate_cache();` here?

Reply:
good catch, its a bad merge, fixed